Backend/ChatAPI.Data/Models/Message.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace ChatAPI.Data.Models;
+
+public class Message
+{
+    public int Id { get; set; }
+    
+    [MaxLength (500)]
```

Comment:
You should use FluentAPI rather than DataAnnotations. It is not wrong to using the DataAnnotations but FluentAPI supports more options and it is more flexible.

Reply:
https://learn.microsoft.com/en-us/ef/core/modeling/ you can see both of them in this documentation.